docs/pages/repo/docs/core-concepts/monorepos/task-graph.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: The Task Graph
+description: Turborepo builds a Task Graph based on your configuration and repository structure.
+---
+
+# The Task Graph
+
+In [the previous section][3], we talked about how Turborepo uses `turbo.json` to express
+how tasks relate to each other. You can think of these relationships as dependencies between
+tasks, but we have a more formal name for them: the task graph.
+
+Turborepo uses a data structure called a [directed acyclic graph (DAG)][1] to
+understand your repository and its tasks. A graph is made up of nodes and edges.
```

Comment:
I'm not sure that I love going as deep as nodes and edges here but I can't defend it since we have a lot of information architecture cleanup to do. Probablyyyy going to get iterated on as we go further.